sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -107,7 +135,7 @@ private[spark] object SQLConf {
         isPublic: Boolean = true): SQLConfEntry[Long] =
       SQLConfEntry(key, defaultValue, { v =>
         try {
-          v.toLong
+          Utils.byteStringAsBytes(v)
```

Comment:
sorry, I thought it was only one place. But actually there are two places. I will create a new method for SHUFFLE_TARGET_POSTSHUFFLE_INPUT_SIZE. Thanks.